src/lib.rs

```diff
@@ -197,6 +230,7 @@ impl fmt::Display for TimedOutError {
 
 impl Error for TimedOutError {}
 
+/// The same API as tokio::time::timeout, except it is dependant on SuspendUnawareInstant for measuring time.
```

Comment:
Instead of dependent, maybe the right word here is "uses"

Reply:
Changing